crates/tako/src/server/monitoring.rs

```diff
@@ -30,6 +31,7 @@ pub enum MonitoringEventPayload {
 
 // Keep the size of the event structure in check
 const _: () = assert!(std::mem::size_of::<MonitoringEventPayload>() == 32);
+const MAX_EVENTS: usize = 10_1000;
```

Comment:
I suggest to set this to `10_000_000`, i.e. 10 million events. Currently, the event record takes `32` bytes, so this should amount to ~300 MiB. @spirali thoughts?

Reply:
This probably should not be a build in but rather a configurable option. I would vote for 100k-1M as default. 10M seems to me as much for a default value.

Reply:
Ok, sounds reasonable. @vyomkeshj please add a CLI parameter to `hq server start` that will configure this, with `1M` as a default value. We can discuss it on MM if you have any questions.

Reply:
on it